src/sssom/util.py

```diff
@@ -1118,20 +1118,20 @@ def _is_iri(string: str) -> bool:
 def get_prefix_from_curie(curie: str) -> str:
     """Get the prefix from a CURIE."""
     if _is_curie(curie):
-        return curie.split(":")[0]
+        return ReferenceTuple.from_curie(curie).prefix
```

Comment:
Cool: https://curies.readthedocs.io/en/latest/api/curies.ReferenceTuple.html

Thanks for that!